OpenKeychain/src/test/java/org/sufficientlysecure/keychain/ssh/signature/SshSignatureConverterTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2017 Christian Hagau <ach@hagau.se>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.sufficientlysecure.keychain.ssh.signature;
+
+import org.bouncycastle.bcpg.PublicKeyAlgorithmTags;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.sufficientlysecure.keychain.KeychainTestRunner;
+
+
+@RunWith(KeychainTestRunner.class)
+public class SshSignatureConverterTest {
+
+    private final static String sCurveOidNistP256 = "1.2.840.10045.3.1.7";
+
+    private final static int[] sRawEcDsaSignatureInt = {
+            0x30, 0x46,
```

Comment:
these can be nicely represented as hex strings with `Hex.decode()`, which also saves the `int[]` hack

Reply:
Ah yes thanks, in hindsight I should have known there would be such a utility function used somewhere in OpenKeychain